include/aws/iot/MqttClient.h

```diff
@@ -29,20 +29,24 @@ namespace Aws
         class AWS_CRT_CPP_API MqttClientConnectionConfig final
         {
           public:
-            MqttClientConnectionConfig() noexcept;
+            static MqttClientConnectionConfig CreateInvalid(int lastError) noexcept;
```

Comment:
why is this function public?

Reply:
MqttClientConnectionConfigBuilder creates the MqttClientConnectionConfig, but isn't friends, so this is public